src/storage/src/window_infer.rs

```diff
@@ -138,10 +141,12 @@ fn align_time_spans_to_windows(durations: &[(i64, i64)], min_duration: i64) -> H
     res
 }
 
-/// Find the most suitable time window size according to the `min_duration` found across all
-/// SST files and memtables through a binary search.
-fn min_duration_to_window_size(min_duration: i64) -> i64 {
-    match TIME_WINDOW_SIZE.binary_search(&min_duration) {
+/// Find the most suitable time window size according to the `min_duration` and `max_duration`
+/// found across all SST files and memtables through a binary search.
+fn duration_to_window_size(min_duration: i64, max_duration: i64) -> i64 {
+    let max_bucket = max_duration >> 7;
```

Comment:
sugg: Make the `7` a constant

Reply:
> sugg: Make the `7` a constant

But I don't know what is the appropriate name of this const. `MAX_WINDOW_PER_FILE_EXP`?